ocaml/xapi/xapi.ml

```diff
@@ -708,6 +708,34 @@ let set_stunnel_timeout () =
   with _ ->
     debug "Using default stunnel timeout (usually 43200)"
 
+(* Consult inventory, because to do DB lookups we must contact the
+ * master, and to do that we need to start an outgoing stunnel. *)
+let set_stunnel_legacy_inv ~__context () =
+  Stunnel.set_good_ciphersuites (match !Xapi_globs.ciphersuites_good_outbound with
+    | "" -> raise (Api_errors.Server_error (Api_errors.internal_error,["Configuration file does not specify ciphersuites-good-outbound."]))
```

Comment:
If the file does not specify `ciphersuites-good-outbound` at all then we use the built-in non-blank default, and we do not raise this error.
The error string here should be "Configuration file specifies a blank string for ciphersuites-good-outbound."